src/tree/SubscriptionTreeItem.ts

```diff
@@ -63,11 +61,8 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
             executeSteps.push(new ResourceGroupCreateStep());
         }
 
-        promptSteps.push(new GitHubOrgListStep());
-        promptSteps.push(new GitHubRepoListStep());
-        if (context.advancedCreation) {
-            promptSteps.push(new GitHubBranchListStep());
-        }
+        // DeploymentMethodStep will add a sub-wizard to determine how to deploy the code that will get run before BuildPresetListStep
+        promptSteps.push(new DeploymentMethodStep());
```

Comment:
What would you think of moving this step first? I just kind of feel like it makes more sense to ask what you're deploying before you ask where you're deploying it to

Reply:
Like before the name step?  It's a little tricky since I'm using subwizards though, because ideally it'd be pick how you're deploying, pick what you're deploying, name, and then all the other steps (like selecting the repo if you are using GitHub or picking a name for the repo if you're publishing to a new one).

How are you envisioning the order?